test/testprocessexecutor.cpp

```diff
@@ -170,6 +177,69 @@ class TestProcessExecutor : public TestFixture {
               "  return 0;\n"
               "}");
     }
+
+    // TODO: provide data which actually shows values above 0
+
+    // TODO: should this be logged only once like summary?
```

Comment:
I think `top5` makes sense in the `summary` and `file` context.

So I suggested we add `top5_file` and `top5_summary` and deprecate `top5`. We could keep `top5` but it currently behaves like `top5_file` so that might be confusing.

Reply:
ok